pkg/demoinfocs/common/equipment.go

```diff
@@ -339,13 +357,30 @@ func (e *Equipment) AmmoInMagazine() int {
 		return -1
 	}
 
-	// need to subtract 1 as m_iClip1 is nrOfBullets + 1
-	return val.IntVal - 1
+	isSource1 := reflect.TypeOf(val.Any).Kind() == reflect.Int
```

Comment:
It's a `uint32` with Source 2, this is a quick way to support both at the same time without a crash